curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657ee28cefc4945568287673.md

```diff
@@ -7,9 +7,15 @@ dashedName: step-3
 
 # --description--
 
-Now that you have an empty list in place, you can start iterating through the input string and start converting each character to snake case.
+With the empty list in place, iterate through the input string and convert each character to snake case using a `for` loop. Iterating over a sequence is called traversal.
 
-Use a `for` loop to iterate through the `pascal_or_camel_cased_string`. Make sure to name the target variable `char` which is short for character. For now, add a `pass` statement in the loop body.
+```python
+# For Loop Example
```

Comment:
It would be OK to remove the syntax of the `for` loop as it is discussed in previous practice projects.